docs/getting-started/user-guides/dcrctl-basics.md

```diff
@@ -0,0 +1,122 @@
+# **dcrctl Basics**
+
+Last updated for v0.8.2.
+
+This guide is intended to help you learn the basic commands of the `dcrctl` application using [startup flags](/getting-started/startup-basics.md#startup-command-flags). 
+
+**Prerequisites:**
+
+- Use either the latest [dcrinstall](/getting-started/install-guide.md#dcrinstall) or [Binary Release archive](/getting-started/install-guide.md#binary-releases) archive to install `dcrctl`.
+- Review how the launch commands for the Command Prompt (Windows) and Bash (OSX/Linux) shells differ [here](/getting-started/operating-systems.md#command-line-differences-across-operating-systems).
+- [Setup dcrd](/getting-started/user-guides/dcrd-setup.md) and have it running in the background.
+- [Setup dcrwallet](/getting-started/user-guides/dcrwallet-setup.md) and have it running in the background.
+
+---
+
+`dcrctl` is the client that controls `dcrd` and `dcrwallet` via RPC commands. You will use `dcrctl` for many things such as checking your balance, buying tickets, creating transactions, and viewing network information.
+
+**REMINDER:** This guide uses OS-agnostic examples for commands. Review the prerequisites to determine if you should be using `./dcrctl` or `dcrctl.exe` instead of `dcrctl`.
+
+---
+
+## Issuing dcrctl Commands
+
+You will need to run dcrctl commands in a separate shell window (Command Prompt/Bash). Commands sent to either `dcrd` or `dcrwallet` will require the username/passwords set during [dcrd Setup](/getting-started/user-guides/dcrd-setup.md) and [dcrwallet Setup](/getting-started/user-guides/dcrwallet-setup.md).
+
+To issue commands to `dcrwallet`, you will need to use `dcrctl -u [wallet username] -P [wallet password] --wallet [command]`.
+
+To issue commands to `dcrd`, you will need to use `dcrctl -u [rpcuser] -P [rpcpass] [command]`.
+
+These passwords can also be stored in a configuration file using this [Storing Login Details](/advanced/storing-login-details/) guide. If stored within configuration files, they do not need to be specified while issuing a `dcrctl` command.
+
+## Unlocking Your Wallet
+
+Some functionality of `dcrwallet` requires the wallet to be unlocked.
+
+The command to unlock your wallet follows: 
+
+```no-highlight
+dcrctl -u [wallet username] -P [wallet password] --wallet walletpassphrase [private passphrase set during wallet creation] 0
+```
+
+Here, we are specifying for `dcrctl` to send the command to `dcrwallet` by using the `--wallet` flag. The `-u` and `-P` flags are specifying the username and password needed to send commands to `dcrwallet`. The actual command is `walletpassphrase` which accepts two parameters, your private passphrase and a time limit. Specifying `0` for a time limit unlocks `dcrwallet` without a time limit. Note, however, that this only unlocks the wallet for the current session. If you close the window the wallet is running in, you will need to unlock it again the next time you start it. 
```

Comment:
You don't have to close the window wallet is running in.  Just stopping and restarting wallet is enough to need to unlock again.